stories/Components/WeeklyPickButton.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Decorator, Meta, StoryObj } from '@storybook/react';
+import { WeeklyPickButton } from '../../components/weekly-pick-button/WeeklyPickButton';
+import { RadioGroup } from '../../components/weekly-pick-button/RadioGroup';
+
+const meta = {
+  title: 'Components/WeeklyPickButton',
+  component: WeeklyPickButton,
+  tags: ['autodocs'],
+  parameters: {
+    docs: {
+      description: {
+        component: 'Button for weekly team selection',
+      },
+    },
+  },
+  argTypes: {
+    team: { description: 'Name of the team' },
+    src: { description: 'Team logo image source' },
+  },
+} satisfies Meta<typeof WeeklyPickButton>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+const withRadioGroup: Decorator = (Story) => (
+  <RadioGroup>
+    <Story />
+    <WeeklyPickButton team="Cowboys" src="/assets/team-cowboys.svg" />
```

Comment:
These team logos should come from the DB. These will not be static.

Reply:
Understood. I had it in there for testing and was expecting them to be deleted once the API is integrated. Should I find something like a plain square as placeholder image?